server/controllers/users.js

```diff
@@ -0,0 +1,78 @@
+import jwt from "jsonwebtoken";
+import users from "../db/users";
+class UsersController {
+  getAllUsers(req, res) {
+    res.status(200).json({
+      status: 200,
+      data: users
+    });
+  }
+
+  signup(req, res) {
+    const {
+      email,
+      firstname,
+      lastname,
+      password,
+      address,
+      status,
+      isAdmin
+    } = req.body;
+
+    const userExists = users.find(user => user.email === email);
+
+    if (userExists) {
+      res.status(400).json({
```

Comment:
1. The status code here is not correct. Check out for the correct status code when user tries to create an account using an email address which already exists in the system.
2. Line 25 should be `return res.status ...`